src/index.ts

```diff
@@ -1,20 +1,20 @@
-import { program } from "commander";
+import { Option, program } from "commander";
 import http from "http";
 
 import { FilesystemCacheBackend } from "./cache";
 import { handleRequest } from "./handleRequest";
 
 program
     .name("swr-cache-proxy")
-    .argument("<origin>", "Origin server URL")
-    .option("--port <port>", "port to listen on", "3000")
-    .option("--cacheSizeLimitHint <megabytes>", "maximum cache size hint", "500")
-    .option("--cacheDir <dir>", "directory cache files will be written into", "cache")
-    .action((origin: string, { port, cacheDir, cacheSizeLimitHint }: { port: string; cacheDir: string; cacheSizeLimitHint: string }) => {
+    .addOption(new Option("--origin <origin>", "Origin server URL").env("ORIGIN_URL").makeOptionMandatory())
```

Comment:
why change that to an option? Imho having this important, required part as argument is a nice cli interface

Reply:
As stated in https://github.com/vivid-planet/swr-cache-proxy/pull/18#issuecomment-1744734689

> I also needed to switch for origin from argument to option to support environment variables.

Alternatively, we could revert this change and use
```
"start": "node dist/index.js $ORIGIN_URL
```